content/apply.md

```diff
@@ -1,42 +1,9 @@
 +++
-title = "Join the Ethical Source Working Group"
-heading = "Sign up to join the Ethical Source working group."
+title = "Join the Movement"
 +++
 
-The Ethical Source working group is a community for open source maintainers and contributors, tech ethicists and advocates, licensing experts, and more to come together to strategize about ways to incorporate ethics into our open source work.
+### Join the Organization for Ethical Source
+If you are interested in joining the Organization for Ethical Source as an active, supporting, or associate member, please [use this form](/apply_org) to get in touch with us. All members of the organization are automatically invited to participate in the Working Group.
```

Comment:
```suggestion
If you are interested in joining the Organization for Ethical Source as an active or support member, please [use this form](/apply_org) to get in touch with us.
```

* Removes the mention to associate member (see rationale below)
* Removes the last sentence that mentions the working group, as it ads confusion; active members will be invited to join the WG, support members won't.